CONTRIBUTING.md

```diff
@@ -47,13 +47,13 @@ a bug in pandoc.  Quoting from the MANUAL:
 > to be perfect, conversions from formats more expressive than pandoc's
 > Markdown can be expected to be lossy.
 
-For example, both docx and odt can represent margin size, but because
-pandoc's internal document model does not contain a representation of
+For example, both `docx` and `odt` formats can represent margin size, but
+because pandoc's internal document model does not contain a representation of
 margin size, this information will be lost on converting from docx
-to odt.  (You can, however, customize margin size using `--reference-odt`.)
+to `odt`.  (You can, however, customize margin size using `--reference-odt`.)
 
 So before submitting a bug report, consider whether it might be
-"out of scope."  If it concerns a feature of documents that isn't
```

Comment:
This is actually the correct punctuation in American style (period inside quote).
It's illogical, I know.